packages/kolibri-tools/.eslintrc.js

```diff
@@ -85,7 +85,7 @@ module.exports = {
       },
     ],
     'vue/attribute-hyphenation': [ERROR, 'never'],
-    'vue/require-default-prop': OFF,
+    'vue/require-default-prop': ERROR,
```

Comment:
You need to revert this line for now until all of the `vue/require-default-prop` errors are fixed. This is because the remaining errors will cause the "GitHub Actions / All file linting" check to fail.
```suggestion
    'vue/require-default-prop': OFF,
```

Reply:
I amended the commit as per you request. If rest is upto your liking I can go ahead and fix all the lint errors.

Reply:
Ok, sounds good. You can fix all of these `'vue/require-default-prop'` errors in this PR, then convert it from a draft to "ready for review" when you're done.